### ISSUE TITLE
STARTTLS vs TLS

### ISSUE THREAD

**Issue by mpking828** [inbucket/inbucket#156]
I'm trying to figure out if I could use inbucket to test my code's compliance with
https://tools.ietf.org/html/rfc8314

tl;dr version
https://kunder.support/knowledgebase/400/SSL-vs-TLS-vs-STARTTLS.html
In the wild, SMTP uses 3 ports:
25 does not typically use any encryption
465 uses Implicit TLS
587 starts plaintext, but upgrades to TLS using STARTTLS

In coding our email client, we of course have to test all three.

Digging thru the code, I see STARTTLS is supported, but I can't tell if Implicit TLS is.

**Comment by jhillyerd** [comment]
No, we don't support implicit TLS.  I think it's worth adding, so will consider this a feature request.

You could try using [OpenSSL stunnel](https://en.wikipedia.org/wiki/Stunnel) to proxy traffic to Inbucket's normal SMTP port to get the same effect.

**Comment by opticoder** [comment]
i can confirm that stunnel works fine with inbucket
